src/hotspot/share/adlc/main.cpp

```diff
@@ -229,6 +229,7 @@ int main(int argc, char *argv[])
   AD.addInclude(AD._CPP_file, "opto/regalloc.hpp");
   AD.addInclude(AD._CPP_file, "opto/regmask.hpp");
   AD.addInclude(AD._CPP_file, "opto/runtime.hpp");
+  AD.addInclude(AD._CPP_file, "runtime/continuation.hpp");
```

Comment:
This adds it for all platforms. Isn't it sufficient to add it in `source %{` section of the ad file?